src/nano_contracts/utils.ts

```diff
@@ -189,6 +215,29 @@ export const validateBlueprintMethodArgs = async (blueprintId, method, args): Pr
           );
         }
         break;
+      // Creating a block {} in the case below
+      // because we can't create a variable without it (linter - no-case-declarations)
+      case 'Address': {
+        const argValue = args[index];
+        if (typeof argValue !== 'string') {
+          throw new NanoContractTransactionError(
```

Comment:
chore: We should add a test to validate those protections from invalid address inputs

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-lib/pull/729/commits/3009dce76ecd371c1e3d1630a58e856b164afb80